kickstarts/partials/post/node.ks.erb

```diff
@@ -0,0 +1,5 @@
+# Get the NodeJS RPM repo setup
+curl --silent --location https://rpm.nodesource.com/setup_6.x | bash -
```

Comment:
Right, I suspect that `.rpm` file contains the repo file which we could just point to directly.